### DISCUSSION TITLE
Setting media path in tina

### DISCUSSION THREAD

**Discussion by KosisoPlato** [tinacms/tinacms#3945] [Help] [answered]
Please I'm having some issues with setting tina media path in my Gatsby project. My media is repo-based (that's it's inside the project). I initially had it in the root folder as `content/images` back in forestry before migration, but now that I've migrated I tried to set the path in the config.js to `publicFolder: "content"` and `mediaRoot: "images"` but I kept getting a value of `images/content/test.png` instead in my frontmatter image source. I even tried moving the images to the actual public folder of Gatsby which is the static folder and then set `publicFolder: "static"` and `mediaRoot: "images"` but I'm still getting `images/content/test.png` in my frontmatter image source after setting the image. Please could any one help and tell me what I'm doing wrong?

**Comment by logan-anderson** [comment]
@KosisoPlato Do you mind sharing a link to your project so I can see the folder structor? 
- Thanks 😄

**Reply by KosisoPlato** [reply]
Sorry, it's a private company project, but here's a screenshot of the folder structure
<img width="886" alt="Screenshot 2023-05-25 at 09 56 43" src="https://github.com/tinacms/tinacms/assets/133699026/038ad12b-cc7a-4701-acc2-205d41eb9efa">
After several troubleshooting I discovered that it's because I initially had my images stored in the md files as `content/images/test.png` that's why it kept updating it as `images/content/test.png` for some reason, but when I added a fresh image, the path now became just `images/test.png`, so if I'm to update all the images I'll get the same path `images/test.png`, but I want the path `content/image/test.png` like I had in forestry or at least `static/images/test.png` like I have in the config, cos Gatsby graphql requires the project root path to generate the images
Here's the result I get after adding an image in the cms 
<img width="385" alt="Screenshot 2023-05-25 at 10 12 47" src="https://github.com/tinacms/tinacms/assets/133699026/bc59608f-3cd7-40e2-b7ca-122b8e48c328">

**Reply by logan-anderson** [reply, marked as answer]
Hello @KosisoPlato the way it works is `publicFolder` is not included in the path so since you have `publicFolder: "static"` it is expected that the path `/images/1.png` is used. If you want `/static/images/1.png` to be the path you can try 

```ts
{
    mediaRoot: "static/images",
    publicFolder: "/",
}
```

For more info you can [check out the docs](https://tina.io/docs/reference/media/repo-based/) feel free to reach out here if you have any more questions.
